### ISSUE TITLE
Could not install py-spy using pip==20.X

### ISSUE THREAD

**Issue by pawel-ziecina** [benfred/py-spy#218]
Could not install py-spy using pip==20.X. On pip==19.X seems to work.

Steps to reproduce:
```
➜  ~ mkvirtualenv new_pip_installer -p $(which python3.6)                                                                                           ∞
Running virtualenv with interpreter /usr/bin/python3.6
Using base prefix '/usr'
New python executable in /home/pawel.ziecina/.virtualenvs/new_pip_installer/bin/python3.6
Also creating executable in /home/pawel.ziecina/.virtualenvs/new_pip_installer/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
➜  ~ workon new_pip_installer                                                                                                                       ∞
➜  ~ pip install --upgrade pip                                                                                                                      ∞
Requirement already up-to-date: pip in ./.virtualenvs/new_pip_installer/lib/python3.6/site-packages (20.0.1)
➜  ~ pip install py-spy                                                                                                                             ∞
ERROR: Could not find a version that satisfies the requirement py-spy (from versions: none)
ERROR: No matching distribution found for py-spy
```

**Comment by pawel-ziecina** [comment]
possible issue with pip as discussed in https://github.com/pypa/packaging/issues/257

**Comment by wiwski** [comment]
I confirm it is a pip issue and it is now fixed in pip version 20.0.2

**Comment by benfred** [comment]
Looks like an issue with pip - closing
